src/main/Sala.java

```diff
@@ -115,7 +124,7 @@ public void intentarIniciarElJuego() {
 		if (esValidoIniciarElJuego()) {
 			generarJuego();
 		} else {
-			notificarActualizacionAJugadores(getCantidadJugadores() - 1);
+			notificarActualizacionAJugadores(getCantidadJugadores());
```

Comment:
Tas seguro que no va el -1?? :s

Reply:
Si el -1 era para que no se fuera de rango, entonces no va, xq el for del método usa < y no "<=". En el otro caso sí iba para no actualizarle al que recién se metía a la sala.
De todas formas no noté diferencia.